scripts/plugins/support-pssl.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2017. All rights reserved.
+
+set -e
+
+# Open/Close port 6640 when pssl is used in ovs
+
+
+OP="$1"
+CHAIN="RH-Firewall-1-INPUT"
+POSITION=$(iptables -L --line-n -n | grep -A 11 "Chain $CHAIN" | grep "443" | cut -d " " -f1)
+RULE="-p tcp -m conntrack --ctstate NEW -m tcp --dport 6640 -j ACCEPT"
+
+case "${OP}" in
+    start)
+        if iptables -S | grep $CHAIN | grep -e "${RULE}" > /dev/null
+        then
+            echo $"port 6640 is opened already"
+        else
+            iptables -I $CHAIN $POSITION $RULE
+            service iptables save
+        fi
+        ;;
+    stop)
+        iptables -D $CHAIN $RULE
+        service iptables save
+        ;;
+    *)
+        echo $"Usage: $0 {start|stop}" 1>&2
```

Comment:
This should be 'open' and 'close' to be clear. I think we should also make this script more generic which should be very simple:

 * Make the port number a parameter to the script.
 * Rename the script to something generic (e.g. firewall-port)
 * Update the XAPI code that calls this script to call `firewall-port open 6640`

This way we have something we can re-use in the future as necessary :)